### PR TITLE
Foreign binary source detection

### PR DESCRIPTION
**Pull request by tomspivak**
@iansk Please see if you can improve the formatting.

### CODE REVIEW

File: admin_guide/runtime_defense/runtime_defense_hosts.adoc [hunk 1]
```diff
@@ -41,19 +41,32 @@ NOTE: The *Prevent* action for detection of file system events requires a Linux
...
-
-- *Processes running from temporary storage* -- Detect processes running from temporary storage (unexpected behavior for legitimate processes). Alert/prevent on file creation or execution.
-
-- *Webshell attacks* -- Detect abuse of web servers vulnerabilities to create a webshell. Alert on webshell creation and and alert or prevent execution of linux command line tools from web servers.
+- *Non-packaged binaries created or run by service* -- Detect binaries created by a service without a package manager.
+Alert on file creation, and alert or prevent their execution.
++
+NOTE: Defender must be running when a file is written to detect its source. 
++
+NOTE: Currently supported only apt, yum, and dnf.
```
Comment (iansk) [inline admin_guide/runtime_defense/runtime_defense_hosts.adoc:49]: @tomspivak This bullet is about non-packaged binaries. However, this note talks about supported package managers. Does this note need to be put somewhere else in this article? Or is there something I'm not understanding?

File: admin_guide/runtime_defense/runtime_defense_hosts.adoc [hunk 2]
```diff
@@ -41,19 +41,32 @@ NOTE: The *Prevent* action for detection of file system events requires a Linux
...
+
+- *Non-packaged binaries created or run by user* -- Detect binaries created by a user without a package manager.
+Alert on file creation, and alert or prevent their execution.
++
+NOTE: Defender must be running when a file is written to detect its source. 
++
+NOTE: Currently supported only apt, yum, and dnf.
```
Comment (iansk) [inline admin_guide/runtime_defense/runtime_defense_hosts.adoc:56]: @tomspivak Same comment as above

**Comment by iansk** [comment]
@tomspivak I fixed the formatting, but I have a couple of questions about the content. Please see my review comments.

File: admin_guide/runtime_defense/runtime_defense_hosts.adoc (hunk 1 above)
Comment (tomspivak) [reply admin_guide/runtime_defense/runtime_defense_hosts.adoc:49]: The location here and below are correct. This feature is about detecting binaries that have been deployed without a package manager, but we depend on the package manager used on the host to be able to detect that, and as such we can only perform the detection if the specified package managers are used.

File: admin_guide/runtime_defense/runtime_defense_hosts.adoc (hunk 2 above)
Comment (tomspivak) [reply admin_guide/runtime_defense/runtime_defense_hosts.adoc:56]: See above

File: admin_guide/runtime_defense/runtime_defense_hosts.adoc (hunk 1 above)
Comment (iansk) [reply admin_guide/runtime_defense/runtime_defense_hosts.adoc:49]: @tomspivak Got it, thanks. I understand now. I'll update your note with this additional context, and then merge your PR.

**Comment by iansk** [comment]
@tomspivak Thanks for the PR. Merging this now.

**Pull request merged**